commands/device/list-applications.ts

```diff
@@ -0,0 +1,20 @@
+///<reference path="../../../.d.ts"/>
+"use strict";
+
+import options = require("./../../options");
+
+export class ListApplicationsCommand implements ICommand {
+	constructor(private $devicesServices: Mobile.IDevicesServices) { }
+
+	allowedParameters: ICommandParameter[] = [];
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			this.$devicesServices.initialize(undefined, options.device, {skipInferPlatform: true}).wait();
```

Comment:
I totally agree with this idea (mostly because this way we'll replace the 
```Typescript
{skipInferPlatform: true}
```
code with just a simple property of the object.